### ISSUE TITLE
Typings for keycloak-connect

### ISSUE THREAD

**Issue by abkaplan07** [DefinitelyTyped/DefinitelyTyped#10130]
We want to use keycloak-connect for our project - would happy to make a typings definition!

**Comment by MumblesNZ** [comment]
@adambkaplan did this ever happen?

**Comment by adambkaplan** [comment]
@MumblesNZ I don't believe so. If you are interested I'd open a github issue on the keycloak/keycloak repo.
